pydantic/functional_validators.py

```diff
@@ -502,7 +502,7 @@ def __call__(  # noqa: D102
 """A `@model_validator` decorated function signature. This is used when `mode='after'`."""
 
 _AnyModelWrapValidator = Union[ModelWrapValidator[_ModelType], ModelWrapValidatorWithoutInfo[_ModelType]]
-_AnyModeBeforeValidator = Union[
+_AnyModelBeforeValidator = Union[
```

Comment:
Good catch - this is confusing bc we do use `mode` as well haha, but `model` is definitely right here.